File: Php/app/User/Listing/UserItemModel.php
namespace app\User\Listing{
	use Domain\Classes\User;
	use app\Urls\UserDetailsUrlModel;

	class UserItemModel{

		public $name;
		public $urlModel;

		public function __construct(User $user){
			name = $user.getDisplayName();
			urlModel = new UserDetailsUrlModel($user);
		}

	}

}

File: Web/Models/UserModels/Listing/UserItemModel.cs
using Core.Classes;
using Web.Models.UrlModels;

namespace Web.Models.UserModels.Listing{
	
    public class UserItemModel{

        public string Name { get; set; }
        public UrlModel UrlModel { get; set; }

		public UserItemModel(User user){
			Name = user.DisplayName;
			UrlModel = new UserDetailsUrlModel(user);
		}

	}

}